commit 54f73f4fc25784595d9983fe38b77adda3144927 (refs/changes/18/4418/2)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2018-06-22T20:03:25+00:00 (1 year, 4 months ago)
    
    TRITON-540 Need to update the ufds-watcher submodule again
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/deps/sdc-napi-ufds-watcher b/deps/sdc-napi-ufds-watcher
index 2d4e757..c41edaa 160000
--- a/deps/sdc-napi-ufds-watcher
+++ b/deps/sdc-napi-ufds-watcher
@@ -1 +1 @@
-Subproject commit 2d4e75711d2212d808bb167b0b3c8d9eecb7d7f1
+Subproject commit c41edaa852414a628e41ae6b133db08e9700ab4b
